Updated Officer info and committee links

AnimeAllstar opened the pull request:
Updated [officer list](https://docs.google.com/spreadsheets/d/1c-CJNhi8fp_BQGBVkyZmH_Yhwx9Piwp7suyheRRgn94/edit?usp=sharing)

kewbish reviewed (commented):
Looks good - should we maybe link the Officers page to the team page directly? It's currently hidden under two menu pages: 1) the 'Misc' at the very bottom of the Executives page, then 2) a Miscellaneous page menu.

AnimeAllstar commented:
> Looks good - should we maybe link the Officers page to the team page directly? It's currently hidden under two menu pages: 1) the 'Misc' at the very bottom of the Executives page, then 2) a Miscellaneous page menu.

I thought about that as well, but I wasn't sure about where to place it.

RamitKataria commented:
Maybe change the "Executives" button on the about page to "Team" and have an "Officers" button either above or below the list of past execs?

eqfy commented:
I think Ramit's suggestion is good. Alternatively we can include a new button `Officers` in between the `Executives` and `Constitutions` button on the About page. This `Officer` button will directly link to the Officer page. I guess Asad you can decide which version you want.

AnimeAllstar commented:
- I added the button in `/about` @eqfy suggested, and I also removed the misc folder since it was empty. 
- Since both officers and executives were in `/about/teams`, I moved them both to their own subfolders. 
- I made a new layout for officers, it's a copy of current-execs but without the pager partial since there is nothing else in the folder and the breadcrumbs are enough.

eqfy commented on layouts/about/current-officers.html:
```diff
@@ -0,0 +1,30 @@
+{{ define "main" }}
+
+{{ $currentOfficers := index (sort .RegularPages "Title" "desc") 0 }}
```
Just curious, does this sorting have any effect to the order of officers shown on the page? Or is the order just determined by the order in the markdown file?

AnimeAllstar replied:
I don't think it affects the order of the officers shown on the page. 

If I had to guess, I think it sorts all markdown files in the folder in descending order of the 'title' field and then sets 'currentOfficers' to the zeroth index in the sorted array.

Since we have only Officers.md in the folder, it doesn't really do much, but since this is just a copy of the current-execs layout, it made sense there since it displayed the most recent execs (current execs).

eqfy reviewed (approved):
Thanks a lot @AnimeAllstar !
